mas/db/ingest_pipeline.sh

```diff
@@ -0,0 +1,140 @@
+set -u
```

Comment:
Shellcheck.net reports the following problems:

```
mas/db/ingest_pipeline.sh:1:1: error: Tips depend on target shell and yours is unknown. Add a shebang. [SC2148]
mas/db/ingest_pipeline.sh:61:9: note: Double quote to prevent globbing and word splitting. [SC2086]
mas/db/ingest_pipeline.sh:63:8: note: Double quote to prevent globbing and word splitting. [SC2086]
mas/db/ingest_pipeline.sh:66:10: note: Double quote to prevent globbing and word splitting. [SC2086]
mas/db/ingest_pipeline.sh:72:8: note: Double quote to prevent globbing and word splitting. [SC2086]
mas/db/ingest_pipeline.sh:75:9: note: Double quote to prevent globbing and word splitting. [SC2086]
mas/db/ingest_pipeline.sh:78:9: note: Double quote to prevent globbing and word splitting. [SC2086]
mas/db/ingest_pipeline.sh:108:5: note: Double quote to prevent globbing and word splitting. [SC2086]
mas/db/ingest_pipeline.sh:108:35: note: Double quote to prevent globbing and word splitting. [SC2086]
mas/db/ingest_pipeline.sh:108:44: note: Double quote to prevent globbing and word splitting. [SC2086]
mas/db/ingest_pipeline.sh:131:6: note: Double quote to prevent globbing and word splitting. [SC2086]
mas/db/ingest_pipeline.sh:131:61: note: Double quote to prevent globbing and word splitting. [SC2086]
mas/db/ingest_pipeline.sh:134:5: note: Double quote to prevent globbing and word splitting. [SC2086]
mas/db/ingest_pipeline.sh:134:36: note: Double quote to prevent globbing and word splitting. [SC2086]
```

Please fix these (you can paste the script into Shellcheck.net to check again.)

Reply:
I added double quotes if the variables are strings. For integers, I leave them no quotes.